<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomScriptsEditor</class>
 <widget class="QMainWindow" name="CustomScriptsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Scripts Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="window">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="header">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="button_CreateNewScript">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Create a new Porymap script file with a default template&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Create New Script...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/images.qrc">
              <normaloff>:/icons/file_add.ico</normaloff>:/icons/file_add.ico</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_LoadScript">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add an existing script file to the list below&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Load Script...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/images.qrc">
              <normaloff>:/icons/file_put.ico</normaloff>:/icons/file_put.ico</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_RefreshScripts">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Refresh all loaded scripts to account for any recent edits&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>Refresh Scripts</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/images.qrc">
              <normaloff>:/icons/refresh.ico</normaloff>:/icons/refresh.ico</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_header">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="button_Help">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/images.qrc">
              <normaloff>:/icons/help.ico</normaloff>:/icons/help.ico</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_ListHeader">
         <property name="text">
          <string>Scripts</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="list">
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDropMode::DragOnly</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::DropAction::IgnoreAction</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::TextElideMode::ElideLeft</enum>
         </property>
         <property name="movement">
          <enum>QListView::Movement::Free</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
